scripts/update-submodule-sources.php

```diff
@@ -32,6 +32,7 @@
 foreach ($vars as $var => $path) {
     $cutNth = 2 + substr_count($path, '/');
 
+    // nosemgrep
```

Comment:
What is the point of this? I assume `shell_exec` is the problem here.

Does Semgrep check M4 files? If not, you can exclude the entire `scripts/` directory.

Either way, the PHP files under `scripts/` shouldn't require scanning.

Reply:
Excluded the `bin` and `scripts` directories as they only contain internal files